server/common/utilities.js

```diff
@@ -6,4 +8,9 @@ const filter = f => object =>
     Object.entries(object).filter(([key, value]) => f(value, key, object))
   );
 
-module.exports = filter;
+const checkRole = (ids, userIdFromReq) => {
```

Comment:
second parameter - just `userId`

Reply:
const userId is used below, function parameter can't be change to userId